website/docs/api/admin/events-api.md

```diff
@@ -217,4 +145,1458 @@ interface IEvent {
+  "environment": null
+}
+```
+
+### `addon-config-deleted`
+
+This event fires when you update an addon configuration.
+The `data` property contains the addon's ID.
+
+``` json title="example event: addon-config-deleted"
+{
+  "id": 964,
+  "type": "addon-config-deleted",
+  "createdBy": "user@company.com",
+  "createdAt": "2022-06-03T10:16:59.723Z",
+  "data": {
+    "id": "2"
+  },
+  "preData": null,
+  "tags": [],
+  "featureName": null,
+  "project": null,
+  "environment": null
+}
+```
+
+
+
+## User events
+
+### `user-created`
+
+This event fires when you create a new user. The `data` property contains the user's information.
+
+``` json title="example event: user-created"
+{
+  "id": 965,
+  "type": "user-created",
+  "createdBy": "user@company.com",
+  "createdAt": "2022-06-03T10:23:47.713Z",
+  "data": {
+    "id": 44,
+    "name": "New User Name",
+    "email": "newuser@company.com"
+  },
+  "preData": null,
+  "tags": [],
+  "featureName": null,
+  "project": null,
+  "environment": null
+}
+```
+
+### `user-updated`
+
+This event fires when you update a user. The `data` property contains the updated user information; the `preData` property contains the previous state of the user's information.
+
+``` json title="example event: user-updated"
+{
+  "id": 967,
+  "type": "user-updated",
+  "createdBy": "user@company.com",
+  "createdAt": "2022-06-03T10:24:26.301Z",
+  "data": {
+    "id": 44,
+    "name": "New User's Name",
+    "email": "newuser@company.com"
+  },
+  "preData": {
+    "id": 44,
+    "name": "New User Name",
+    "email": "newuser@company.com"
+  },
+  "tags": [],
+  "featureName": null,
+  "project": null,
+  "environment": null
+}
+```
+
+### `user-deleted`
+
+This event fires when you delete a user. The `preData` property contains the deleted users information.
+
+``` json title="example event: user-deleted"
+{
+  "id": 968,
+  "type": "user-deleted",
+  "createdBy": "user@company.com",
+  "createdAt": "2022-06-03T10:24:49.153Z",
+  "data": null,
+  "preData": {
+    "id": 44,
+    "name": "New User's Name",
+    "email": "newuser@company.com"
+  },
+  "tags": [],
+  "featureName": null,
+  "project": null,
+  "environment": null
+}
+```
+
+
+
+## Environment events
+
+### `environment-import`
+
+This event fires when you import an environment (custom or otherwise) as part of an import job. The `data` property contains the configuration of the imported environment.
+
+``` json title="example event: environment-import"
+{
+  "id": 24,
+  "type": "environment-import",
+  "createdBy": "import-API-token",
+  "createdAt": "2022-06-03T11:30:40.557Z",
+  "data": {
+    "name": "custom-environment",
+    "type": "test",
+    "sortOrder": 9999,
+    "enabled": true,
+    "protected": false
+  },
+  "preData": null,
+  "tags": [],
+  "featureName": null,
+  "project": null,
+  "environment": null
+}
+```
+
+
+
+### `drop-environments`
+
+This event fires when you delete existing environments as part of an import job.
```

Comment:
Does the data here always contain "all-projects"? If so it might be nice to include that in the description

Reply:
Update: yes, all drop events say "all-x", where "x" is whatever resource they're dropping. Save for "drop-environments", which says "all-projects". I think that is an oversight, so I've changed the source code for that too.

Reply:
Addressed in 2d85299c339ceb83513f8290ecfcb7fa46162f99